athena-federation-integ-test/src/main/java/com/amazonaws/athena/connector/integ/IntegrationTestBase.java

```diff
@@ -51,11 +52,10 @@
     private static final String ATHENA_QUERY_RUNNING_STATE = "RUNNING";
     private static final String ATHENA_QUERY_FAILED_STATE = "FAILED";
     private static final String ATHENA_QUERY_CANCELLED_STATE = "CANCELLED";
-    private static final String ATHENA_FEDERATION_WORK_GROUP = "AmazonAthenaPreviewFunctionality";
+    private static final String ATHENA_FEDERATION_WORK_GROUP = "FederationIntegrationTests";
```

Comment:
can we make this configurable with a good default like this one?

Reply:
will include as parameter in config file with the value above as default.